tests/integration/update.yaml

```diff
@@ -36,12 +36,6 @@ policy:
     with:
       trust_root: sse
       delegations: ["belitzphilipp", "starkteetje"]
-  - pattern: "docker.io/securesystemsengineering/connaisseur:*"
-    validator: dockerhub
-    with:
-      trust_root: sse
-  - pattern: "docker.io/securesystemsengineering/connaisseur:helm-hook-*"
-    validator: allow
```

Comment:
this configuration seems to be bound to fail. You removed the connaisseur image pattern and only add an allow for the `cicd.yaml` which means that there is no pattern for the `release.yaml`. I would propose to keep the entry with `allow` entry for cicd and overwrite the entry using `yq` to the dockerhub sse validator for release or the other way round. Only challenge is you will have to pick the correct entry by number as it is an array which will fail at some point when we change the pre-configuration...
So maybe choose option three and add an entry during the `release.yaml` as well...

Reply:
i don't quite understand, the `cicd.yaml` pipeline and the `release.yaml` pipeline run independently from eachother, so changes to `cicd.yaml` do not affect `release.yaml`. and for `release.yaml`, there is still a `*:*` rule, that the connaisseur image applies to.

Reply:
you are correct. did not see that the default policy is sse pub key